data.js

```diff
@@ -0,0 +1,136 @@
+export const projects = {
+  caveat: {
+    name: "Caveat",
+    images: {
+      large: "/images/6_Hellocaveat.png",
+    },
+    logo: "/images/Caveat_White.svg",
+    href: "/info/caveat",
+    color: "blue",
+    title:
+      "Front-End, Back-End and support for the website of the creative studio Caveat",
+    url: "hellocaveat.com",
+    address: "New York, US",
+    header: "Website maintenance of creative studio Caveat",
+    stackTitle: "Stack: ",
```

Comment:
it isn't changeable
so, let's remove it from here and put it inside "code"

in this file we put only info, that depends from each project

Reply:
here i am confused. I understand about stackTitle, but I have no idea how to fix url, address and header